app/src/main/java/eu/depau/etchdroid/services/job/dto/JobServiceIntentDTO.kt

```diff
@@ -1,13 +1,14 @@
 package eu.depau.etchdroid.services.job.dto
 
 import android.os.Parcel
+import android.os.Parcelable
 import eu.depau.kotlet.android.parcelable.KotletParcelable
-import eu.depau.kotlet.android.parcelable.parcelableCreator
 import java.io.Serializable
 
 data class JobServiceIntentDTO(
         val jobId: Long
 ) : KotletParcelable, Serializable {
```

Comment:
`Missing Parcelable CREATOR field`

Reply:
Cause it has to be `@JvmField` and `@JvmStatic` or something like that.

Reply:
In general, revert this and any changes to KotlinParcelable cause this worked perfectly fine before. **Do not fix what's clearly not broken**.

Reply:
Try to set it as `@JvmField` and there still be a warning